DiamondLightSource/SynchWeb
# LIMS-1351: Switch to simpler arrows

**JIRA ticket**: [LIMS-1351](https://jira.diamond.ac.uk/browse/LIMS-1351)

**Summary**:

Arrows introduced in https://github.com/DiamondLightSource/SynchWeb/pull/735 don't work in Safari or Firefox, so switch to simpler icons.

**Changes**:
- Replace &#11167; with &darr;
- Replace &#11165; with &uarr;

**To test**:
- Go to a live visit (eg cm37235-3) and look at the Show Webcams and Beamline Status link at the top, it should end with a down arrow.
- Click the link, the down arrow should become an up arrow.
